sarsen/radiometry.py

```diff
@@ -18,14 +18,13 @@ def sum_weights(
     multilook: T.Optional[T.Tuple[int, int]] = None,
 ) -> xr.DataArray:
     geocoded = initial_weights.assign_coords(
-        slant_range_index=slant_range_index, azimuth_index=azimuth_index
+        # Use z_level names for backward compatibility with xarray<=2022.03.0
+        z_level_0=slant_range_index,
+        z_level_1=azimuth_index,
```

Comment:
I'm pretty sure this is needed when the `multilook` option is turned on. `xarray<=2022.03.0` and therefore `sarsen` uses fixed names ("z_level_0", "z_level_1"), whereas the latest version of xarray would use ("slant_range_index", "azimuth_index").

I don't think `multilook` is tested by the CI though, so I haven't really tested this.